topic_dynamics/slicing.py

```diff
@@ -0,0 +1,37 @@
+"""
+The functionality of the temporal slicing of projects.
+"""
+
+from datetime import datetime, timedelta
+import os
+from typing import List
+
+
+def get_dates(n_dates: int, time_delta: int) -> List:
+    """
+    Creates a list of a given number of the datetime objects with a given step.
+    :param n_dates: the amount of dates.
+    :param time_delta: the time step between dates
```

Comment:
```suggestion
    :param day_delta: the number of days between dates.
```
I think it better describes the logic of function.